webapi/Skills/ChatSkills/CopilotChatPlanner.cs

```diff
@@ -45,10 +47,16 @@ public class CopilotChatPlanner
     /// Regex to match variable names from plan parameters.
     /// Valid variable names can contain letters, numbers, underscores, and dashes but can't start with a number.
     /// Matches: $variableName, $variable_name, $variable-name, $some_variable_Name, $variableName123, $variableName_123, $variableName-123
-    /// Does not match: $123variableName, $100 $200 
+    /// Does not match: $123variableName, $100 $200
     /// </summary>
     private const string VARIABLE_REGEX = @"\$([A-Za-z]+[_-]*[\w]+)";
 
+    /// <summary>
+    /// Supplemental text to add to the plan goal if PlannerOptions.Type is set to Stepwise.
+    /// Helps the planner know when to bail out to request additional user input.
+    /// </summary>
+    private const string STEPWISE_PLANNER_SUPPLEMENT = "If you need more information to fulfill this request, return with a request for additional user input.";
```

Comment:
Nit: in C#, constants normally just use PascalCase. Same with VariableRegex

Reply:
Good to know, will get these changed!